mix.exs

```diff
@@ -5,7 +5,7 @@ defmodule ExCrypto.Mixfile do
     [app: :ex_crypto,
      version: "0.5.0",
      name: "ExCrypto",
-     elixir: ">= 1.0.0",
+     elixir: ">= 1.3.0",
```

Comment:
Does this PR use any functions that require Elixir 1.3 or higher? If so that's cool, if not I'd like to leave this requirement where it is so the library works with projects that haven't migrated to the newer version yet.